Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/Forme.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef FORME_HPP_2023
```

Comment:
"#pragma once" should be enough because it allows to not include the file multiple times

Reply:
Même si on migre vers CMake ? Je pensais que c'était un special VS
Au passage, j'avais pas remarqué mais les fichiers sont en .h e non .hpp (j'ai pas fait gaffe en les créant) je les renommerai